index.js

```diff
@@ -68,11 +73,15 @@ let definitionCommand = require('./commands/definition')();
 let goodMorningGreetingCommand = greetingCommand(channelIdForGeneral, iconUrl, bot);
 let goodNightGreetingCommand = greetingCommand(channelIdForGeneral, iconUrl, bot, `Good night ${appName}`);
 let getAstronomyPictureOfTheDayCommand = require('./commands/get_astronomy_picture_of_the_day')(channelIdForRandom, iconUrl, bot);
+let announceEventsHappeningTodayCommand = require('./commands/announce_events_happening_today')(channelIdForEvents, iconUrl, bot);
+let announceEventsHappeningTomorrowCommand = require('./commands/announce_events_happening_tomorrow')(channelIdForEvents, iconUrl, bot);
 
 // scheduled messages
 schedule.scheduleJob('30 9 * * *', goodMorningGreetingCommand);
```

Comment:
Good point, and I think `scheduler` is a better fit. But the Node package in use is called `node-schedule` which makes me thing that's where the variable's name is coming from. Also I don't know what's the popular variable name when it's `require`d. Anyone used it in their project? I have zilch Node experience 😞